plasma_framework/contracts/src/framework/BlockController.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^0.5.0;
+
+import "./models/BlockModel.sol";
+import "./registries/VaultRegistry.sol";
+import "./utils/Operated.sol";
+
+contract BlockController is Operated, VaultRegistry {
+    uint256 private _childBlockInterval;
+    uint256 private _nextChildBlock;
+    uint256 private _nextDepositBlock;
+
+    mapping (uint256 => BlockModel.Block) private _blocks;
```

Comment:
why do you want to keep this private?

Reply:
I am on the fence for this file actually. 

In general I prefer private so it promises any contract inherits this cannot accidentally set the value. So, for instance, the `operator` field for `Operated.sol` makes perfect sense to be private as it is inherited in several places and in deep layer. However, on the other hand, If it is a top level contract code that nobody inherits, there is almost no benefit to keep private as nothing is going to inherit it.

The case here to me is in a gray area. It is designed to be inherit by `PlasmaFramework` but functional wise, it should be the top layer code here. `PlasmaFramework` should simply only extends several modules to glue things into one contract. In this case, I chose to see how hard it would be to keep private in coding and testing. Since it is not too much more complex I chose private here. But for the case here I am pretty open to make it public.

Reply:
Calling a method is rather more expensive than just retrieving the stored data, but now I understand your decision I think that for now it can be as it is :)